src/elements/hx-tabcontent/index.spec.js

```diff
@@ -0,0 +1,75 @@
+import { fixture, oneEvent, expect } from '@open-wc/testing';
+
+/**
+ * <hx-tabcontent> component tests
+ *
+ * @type HXTabcontentElement
+ *
+ */
+describe('<hx-tabcontent> component tests', () => {
```

Comment:
### Your tests should fail based on this error, otherwise it's not validating/testing what we need to resolve Issue #516.

<img width="1195" alt="Screen Shot 2020-06-17 at 7 59 02 PM" src="https://user-images.githubusercontent.com/10120600/84965641-0e243d00-b0d5-11ea-8484-1d44efa77470.png">

Reply:
This error maybe caused when the parent hx-tabset does not have an ID. We fixed this same error recently by adding ID's to all out tabsets. it looks like helix is supposed to add a generated id to the parent tabset, but in React, that generated ID is not present soon enough before the child tabs render, so we get the above error. Possibly a more specific error might be helpful to users to resolve it more easily such as: "Parent hx-tabset element is required to have an id", or maybe the error can get caught or handled, as our tabs seemed to work, even though we got these error. I don't know enough about the underlying code to make a good suggestion, just passing out our experience incase its useful. When we work on the tabs components for Helix React we may be able to create fix / work around inside helix react as well.

Reply:
@nicko-winner @BY00565233, I was able to replicate the above failures with our component testing pipeline:

<img width="959" alt="Screen Shot 2020-07-28 at 5 48 17 PM" src="https://user-images.githubusercontent.com/10120600/88729936-95b09300-d0fa-11ea-9d51-4e0db54324ce.png">

@BY00565233, please add the skipped tests referenced in PR #723.  Once the dynamic tab bug patch is implemented, we will include these tests to our test run.